packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.tsx

```diff
@@ -33,14 +37,15 @@ const PaymentAgentList = observer(({ setSideNotes }: TProps) => {
         } else {
             setSideNotes?.([]);
         }
-    }, [setSideNotes, general_store.is_loading, payment_agent.is_try_withdraw_successful]);
+    }, [setSideNotes, general_store.is_loading, payment_agent.is_try_withdraw_successful, current_language]);
 
     return (
         <div className='payment-agent-list cashier__wrapper--align-left'>
             <div
                 className={classNames('payment-agent-list__instructions', {
                     'payment-agent-list__instructions-hide-tabs': payment_agent.is_try_withdraw_successful,
                 })}
+                key={current_language}
```

Comment:
@george-usynin-deriv why do we need this key here? we are not looping over anything :thinking:

Reply:
This will re-render everytime the language changes right?, is this intended ? 🤔

Reply:
Yup it's needed, Since the components are wrapped with `observer` and `observer` uses `useMemo` internally it will avoid rerendering when the language changes, So we have to use a key to force rerender it when the language changes.

Reply:
thanks guys :pray: